lib/gears/table.lua

```diff
@@ -165,6 +165,21 @@ function gtable.keys(t)
     return keys
 end
 
+--- Get the number of keys in a table, both integer and string indicies.
+--
+-- This is functionally equivalent, but faster than `#gears.table.keys(t)`.
+--
```

Comment:
Can you add an `@DOC_..._EXAMPLE@` here? Copy paste a `tests/examples/text/gears/table/<something>.lua` file with a new name, then reference it here.

I/we are trying to enforce that all new APIs have an unit-tested code example these days. Think the of "StackOverflow syndrome".  People tend to copy paste code even if they actually know how to write it :P